lnwallet/interface_test.go

```diff
@@ -1669,7 +1672,8 @@ func testPublishTransaction(r *rpctest.Harness,
 	tx1 := newTx(t, r, keyDesc.PubKey, alice, false)
 
 	// Publish the transaction.
-	if err := alice.PublishTransaction(tx1); err != nil {
+	err = alice.PublishTransaction(tx1, labels.External)
```

Comment:
Any reason to provide this label for the tests?

Reply:
Just personal preference to not have a bunch of blank fields all over tests, makes them a bit less readable imo. Happy to remove if necessary.